### PR TITLE
Fix typo in phrase 'too low'

### PR DESCRIPTION
**Pull request by michbil**

### CODE REVIEW

File: test/api.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ tape('[Transaction]: Basic functions', function (t) {
 
   t.test('should give a string about not verifing Signatures', function (st) {
     transactions.forEach(function (tx) {
-      st.equals(tx.validate(true).slice(0, 53), 'Invalid Signature gas limit is to low. Need at least ')
+      st.equals(tx.validate(true).slice(0, 53), 'Invalid Signature gas limit is too low. Need at least ')
```
Comment (fanatid) [inline test/api.js]: hehe, you did not fix tests here
should be `.slice(0, 54)` now

**Comment by fanatid** [comment]
squash commits please

**Comment by michbil** [comment]
@fanatid should be ok now

**Comment by fanatid** [comment]
Thank you!

**Pull request merged**